src/add-api/index.md

```diff
@@ -0,0 +1,31 @@
+# How do I add scoreboards and trophies to my game?
+
+First, you need to implement the Game API in your game's code. Then, you can add scoreboards and trophies to your game.
+
+## Scoreboards
```

Comment:
This is what I have so far: https://docs.google.com/document/d/1MCie-1C8vlBztRkxYbnqqN31YUuqJkVqqLvOyuJJozc/edit